src/main/java/com/mixpanel/android/viewcrawler/ViewVisitor.java

```diff
@@ -81,6 +101,8 @@ protected String name() {
 
         private final Caller mMutator;
         private final Caller mAccessor;
+        private final WeakHashMap<View, Object> mOriginalValues;
```

Comment:
Should we change this guy to Map, as you said "We always try to use the abstract interfaces "?